text/0240-Sampling_Configuration.md

```diff
@@ -130,6 +130,56 @@ where each RULE is
 The Predicates represent logical expressions which can access Span Attributes (or anything else available when the sampling decision is to be taken), and perform tests on the accessible values.
 For example, one can test if the target URL for a SERVER span matches a given pattern.
 
+## Example
+
+Let's assume that a user wants to configure head sampling as follows:
+- for root spans:
+    - drop all `/healthcheck` requests
+    - capture all `/checkout` requests
+    - capture 25% of all other requests
+- for non-root spans
+    - follow the parent sampling decision
+    - however, capture all calls to service `/foo` (even if the trace will be incomplete)
+- in any case, do not exceed 1000 spans/second
+
+Such sampling requirements can be expressed as:
+```yaml
+  samplerType: AllOf
+  parameters:
+  - - samplerType: AnyOf
```

Comment:
is the extra dash intentional here (and below)?

Reply:
Yes, we have nested lists here, the list of samplers is an element of the parameter list.

Reply:
Not applicable anymore, the new syntax mostly avoids these nested lists.